Data-Structure/Collection-In-Java/CollectionOfEmployee/TestEmployee.java

```diff
@@ -0,0 +1,92 @@
+package CollectionOfEmployee;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class TestEmployee{
+	private Company company;
+	
+	@Before
+	public void setUp(){
+		company = new Company();
+	}
+	
+	@Test
+	public void addEmployee_HappyFlow(){
+		assertTrue(company.addEmployee("2", "Jai", "Jaipur"));
+		assertTrue(company.addEmployee("3", "Niraj", "Ajmer"));
+		assertTrue(company.addEmployee("1", "Abhay", "Jodhpur"));
+		assertTrue(company.addEmployee("5", "Twinkle", "Sikar"));
+		assertTrue(company.addEmployee("4", "Namrata", "Delhi"));
+	}
+	
+	@Test(expected = AssertionError.class)

```

Comment:
if first assertion will give error then the next one will never be executed. 
Also, use try catch to ensure that the method actually throws expected exception